### DISCUSSION TITLE
[@mantine/spotlight] add ReactNode as supported type for SpotlightAction description

### DISCUSSION THREAD

**Discussion by tylertyssedal** [mantinedev/mantine#4097] [Feature requests]
Currently the SpotlightAction interface has the following for `description`: 

```
    /** Action description, small text displayed after title, used for default filtering */
    description?: string;
```

In testing, it seems to work well with passing in an element/ReactNode. IDE throws a type error, but the component renders properly. This allows for a more controlled description.

**Comment by rtivital** [comment]
Description is used for search, if it is not string then it cannot be searched

**Reply by tylertyssedal** [reply]
I wrote a custom filter to get around this, using the keywords. 

It seems like a worthwhile option, considering it functionally works at the offset of needing to write a custom filter (which I needed to do anyway).
